### PR TITLE
Add Quefrency Rev 2

### PR DESCRIPTION
**Pull request by nooges**
## Description

Adds support for Quefrency Rev. 2

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/quefrency/rev2/config.h [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+#define DEVICE_VER      0x0200
+#define MANUFACTURER    Keebio
+#define PRODUCT         Quefrency
+#define DESCRIPTION     Split 60/65 percent staggered keyboard
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 9
+
```
Comment (drashna) [inline keyboards/keebio/quefrency/rev2/config.h]: Need a diode direction specified. 

```suggestion

#define DIODE_DIRECTION COL2ROW
```

File: keyboards/keebio/quefrency/rules.mk [hunk 2]
```diff
@@ -31,5 +31,3 @@ RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight.
 SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
 
 SPLIT_KEYBOARD = yes
-
-DEFAULT_FOLDER = keebio/quefrency/rev1
```
Comment (drashna) [inline keyboards/keebio/quefrency/rules.mk]: Removing this is making travis CI fail. 

Basically this needs to be re-added, or all this stuff moved out of the quefrency folder, and into the specific revisions.

Comment (nooges) [reply keyboards/keebio/quefrency/rules.mk]: Alright, I'll just revert for now.

**Comment by drashna** [comment]
Travis CI errors are now due to config overlap (redefines and such), and should be okay.

File: keyboards/keebio/quefrency/rev2/rev2.c [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#include "quefrency.h"
+
+void matrix_init_kb(void) {
+	matrix_init_user();
+};
```
Comment (fauxpark) [inline keyboards/keebio/quefrency/rev2/rev2.c]: This function can be deleted.

File: keyboards/keebio/quefrency/rev2/rev2.h [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+    { LD1, LD2, LD3, KC_NO, LD5, LD6, LD7, LD8, LD9 }, \
+    { LE1, LE2, LE3, LE4, LE5, LE6, KC_NO, LE8, KC_NO }, \
+    { RA1, RA2, RA3, RA4, RA5, RA6, RA7, RA8, RA9 }, \
+    { RB1, RB2, RB3, RB4, RB5, RB6, RB7, RB8, RB9 }, \
+    { RC1, RC2, RC3, RC4, RC5, RC6, KC_NO, RC8, RC9 }, \
+    { RD1, RD2, RD3, RD4, RD5, KC_NO, RD7, RD8, RD9 }, \
+    { RE1, RE2, KC_NO, RE4, RE5, RE6, RE7, RE8, RE9 } \
+  }
+
+#define LAYOUT_60_ISO( \
```
Comment (noroadsleft) [inline keyboards/keebio/quefrency/rev2/rev2.h]: Layout macro names should be, with the exception of the `LAYOUT` prefix, entirely lowercase.

```suggestion
#define LAYOUT_60_iso( \
```

File: keyboards/keebio/quefrency/rev2/rev2.h [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+    { KC_NO, KC_NO, LD3, LD4, LD5, LD6, LD7, LD8, LD9 }, \
+    { KC_NO, KC_NO, LE3, LE4, LE5, LE6, KC_NO, LE8, KC_NO }, \
+    { RA1, RA2, RA3, RA4, RA5, RA6, RA7, RA8, KC_NO }, \
+    { RB1, RB2, RB3, RB4, RB5, RB6, RB7, KC_NO, KC_NO }, \
+    { RC1, RC2, RC3, RC4, RC5, RC6, RC7, RC8, KC_NO }, \
+    { RD1, RD2, RD3, RD4, RD5, KC_NO, RD7, RD8, KC_NO }, \
+    { RE1, RE2, KC_NO, RE4, RE5, RE6, RE7, RE8, KC_NO } \
+  }
+
+#define LAYOUT_65_ISO( \
```
Comment (noroadsleft) [inline keyboards/keebio/quefrency/rev2/rev2.h]:
```suggestion
#define LAYOUT_65_iso( \
```

File: keyboards/keebio/quefrency/rev2/rev2.h [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+    { KC_NO, KC_NO, LD3, LD4, LD5, LD6, LD7, LD8, LD9 }, \
+    { KC_NO, KC_NO, LE3, LE4, LE5, LE6, KC_NO, LE8, KC_NO }, \
+    { RA1, RA2, RA3, RA4, RA5, RA6, RA7, RA8, RA9 }, \
+    { RB1, RB2, RB3, RB4, RB5, RB6, RB7, KC_NO, RB9 }, \
+    { RC1, RC2, RC3, RC4, RC5, RC6, RC7, RC8, RC9 }, \
+    { RD1, RD2, RD3, RD4, RD5, KC_NO, RD7, RD8, RD9 }, \
+    { RE1, RE2, KC_NO, RE4, RE5, RE6, RE7, RE8, RE9 } \
+  }
+
+#define LAYOUT_60_ISO_with_macro( \
```
Comment (noroadsleft) [inline keyboards/keebio/quefrency/rev2/rev2.h]:
```suggestion
#define LAYOUT_60_iso_with_macro( \
```

File: keyboards/keebio/quefrency/rev2/rev2.h [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+    { LD1, LD2, LD3, LD4, LD5, LD6, LD7, LD8, LD9 }, \
+    { LE1, LE2, LE3, LE4, LE5, LE6, KC_NO, LE8, KC_NO }, \
+    { RA1, RA2, RA3, RA4, RA5, RA6, RA7, RA8, KC_NO }, \
+    { RB1, RB2, RB3, RB4, RB5, RB6, RB7, KC_NO, KC_NO }, \
+    { RC1, RC2, RC3, RC4, RC5, RC6, RC7, RC8, KC_NO }, \
+    { RD1, RD2, RD3, RD4, RD5, KC_NO, RD7, RD8, KC_NO }, \
+    { RE1, RE2, KC_NO, RE4, RE5, RE6, RE7, RE8, KC_NO } \
+  }
+
+#define LAYOUT_65_ISO_with_macro( \
```
Comment (noroadsleft) [inline keyboards/keebio/quefrency/rev2/rev2.h]:
```suggestion
#define LAYOUT_65_iso_with_macro( \
```

**Comment by noroadsleft** [comment]
Just a note, @bcat's keymap doesn't compile with these changes due to an `RGBLED_SPLIT` conflict.

**Comment by noroadsleft** [comment]
Thanks!

**Comment by bcat** [comment]
Yeah no worries. I'll fix the keymap to `#undef RGBLED_SPLIT`, but I only have a rev1 anyway, so no big loss. :)

**Pull request merged**